### PR TITLE
Read property page

### LINKED ISSUE #54: Search for Properties - Display all the available properties according to User Type
Setup "for rent" and "for sale" pages by showing each house in a square template
Have them connected to the database of houses

### LINKED ISSUE #88: CRUD on properties - Display broker's managed properties

### PR DESCRIPTION
**Pull request by SingingPhantom**
The application can now fetch all the available properties in the database and displays them. The broker can now click on "Manage Properties" to view all properties managed by them. Both tasks have similar implementations.

closes #54, closes #88

### CODE REVIEW

File: UrbanGate/Frontend/src/assets/Components/_Broker/ManageProperties.tsx
Comment (MatthewTP0) [inline UrbanGate/Frontend/src/assets/Components/_Broker/ManageProperties.tsx:1]: Recycling the code to display properties from Houses.tsx and implementing it into the ManageProperties.tsx for the brokers was done very well and is efficient from a time perspective. Looks very good.

File: UrbanGate/Frontend/src/assets/Components/_Houses/Houses.tsx
Comment (MatthewTP0) [inline UrbanGate/Frontend/src/assets/Components/_Houses/Houses.tsx:1]: This file has conflict with the main branch and is preventing the merge. I compared the two documents and I believe it is safe to overrwite the Houses.tsx file in main with the Houses.tsx file here. No old code seems to be needed to be carried over to the new file.

File: UrbanGate/Frontend/src/main.tsx
Comment (MatthewTP0) [inline UrbanGate/Frontend/src/main.tsx:1]: The routes for the broker page are very organized and properly nested within the broker page. Well done.

File: UrbanGate/Frontend/src/assets/Components/_Houses/Houses.tsx
Comment (MatthewTP0) [inline UrbanGate/Frontend/src/assets/Components/_Houses/Houses.tsx:1]: The changes on this file are all very good, they simplify and reorganize previous code to improve readability aswell as adds the axios function that interacts with the backend.
I am going to approve the merge. Excellent work!

**Review by Mohamed-Redouane — APPROVED** [review]

**Review by mylinhtang — APPROVED** [review]

**Review by Azmi-21 — APPROVED** [review]

**Review by MatthewTP0 — APPROVED** [review]

**Pull request merged**